### DISCUSSION TITLE
Enable pencil only option

### DISCUSSION THREAD

**Discussion by smnaveen96** [keiwando/toonsquid#96] [Ideas]
Previously you introduced disable/enable finger drawing option.. but the problem is i can still able to use pen tool by fingers.. which is somewhat not intuitive.. if you see any app for example in procreate if you disable finger drawing.. you cant use selection tool by fingers.. artstudio pro has pencil only mode.. and the same case in affinity.. you cant use shape/node/pen tool by fingers.. 

To sum it up.. fingers only supposed to be used for gestures and selecting tools/layers…

@keiwando

**Comment by keiwando** [comment]
Do you just want the finger to not be able to add new path control points or not even edit exiting ones? And what about the transform tool?

**Reply by smnaveen96** [reply]
Yes.. just selecting tools/layers is enough.. editing using fingers is not intutive.. we can do those performs precisely using pencil.. every other software follows this like procreate, affinity , artstudio  etc…

**Reply by keiwando** [reply]
The first example in your list (Procreate) already doesn't have the behaviour you describe.

**Reply by smnaveen96** [reply]
We can change the settings in procreate.. right now i use fingers only for selecting tools and layers and performing gestures like undo , three fingers swipe down…. Other than that i do everything using pencil..

**Reply by keiwando** [reply]
I'll consider adding the option to disable all editing with the finger as a separate setting in the future but I don't agree that a setting to "Disable finger drawing" should control all of these other editing behaviours as well. It clearly doesn't work the same way in every app either.

**Reply by keiwando** [reply]
Which Procreate settings do you use to disable the finger editing in the transform tool or the ability to edit QuickShape control points with your finger?

**Comment by smnaveen96** [comment]
![75895FA9-E681-4CAB-8CA2-A9DF572083F5](https://user-images.githubusercontent.com/65828780/209534114-e1690107-a309-4672-a6b2-b8c533650fb6.png)

in procreate settings > preference > Gesture controls >  general .. in that general setting there is option on top called enable painting with finger.. if you enable it you can paint using fingers and if you select selection you can select the pixels in the layer you are working.. like you literally can use free freeform selection and make selection using your fingers.. if you disable painting with finger option.. you can either draw using finger and make selection using your fingers as well..

**Reply by keiwando** [reply]
Right, that's the setting I'm referring to as well. But disabling this does not disable your ability to still edit QuickShape control points or use the transform tool with your fingers. In Vectornator for example, the single setting disables all of these interactions, which is what you were asking for. That's why I said Procreate does not seem to be an example for the behaviour that you requested

**Comment by smnaveen96** [comment]
Yes sorry.. my bad.. you were right…  yes i was referring like in vectornator..
